src/lindistflow/extend.jl

```diff
@@ -60,43 +60,43 @@ function LDF.constrain_loads(m::JuMP.AbstractModel, p::LDF.Inputs, ps::Array{REo
     for j in p.busses
         if j in keys(p.Pload)
             if parse(Int, j) in reopt_nodes
-                @constraint(m, [t in 1:p.Ntimesteps],
+                @constraint(m, [t in 1:p.Ntime_steps],
```

Comment:
this is a breaking change. (The p here is LDF.Inputs, not REoptInputs). I will fix it.